README.md

```diff
@@ -37,6 +37,30 @@ signatures:
 
 This policy will validate all images with the public keys provided, and images whose name matches `ghcr.io/kubewarden/*` with the keyless provided.
 
+Another example that validates a GitHub action:
+
+```
+signatures:
+- image: "ghcr.io/kubewarden/*"
```

Comment:
mh, just realized. This doesn't make any mention to `githubActions`. We should nest `owner` and `repo` under something similar, as it happens with `keyless_prefix`.

Reply:
we would to add the GithubActions struct and then change it in the [sdk](https://github.com/kubewarden/policy-sdk-rust/blob/main/src/host_capabilities/mod.rs#L57)

I think it is a good idea, because in the future we might have more (e.g gitlab..)

Reply:
I think you are right, and it's better added there. While we are at it, we could make it a vector, the same way `keyless` and `keyless_prefix` are. This also makes me think, and desire for VerificationConfig V2 in the future supporting `image: '*'` and so.

Reply:
Opened https://github.com/kubewarden/policy-sdk-rust/pull/51, if that's acceptable, we can release, consume on policy-evaluator and make another release of policy-evaluator.

Reply:
@viccuad why would someone need to specify multiple GitHub signatures? I'm missing the use case

Reply:
I think you are right it has no usecase. Adding several owners, the image would be need to signed by all of them, which isn't possible :sweat: .

Reply:
agh. We didn't do this, nesting `owner` and `repo` under a `github_actions` of sort..

Reply:
done in https://github.com/kubewarden/policy-sdk-rust/pull/52/commits/4ae83b109a712b69085e187eea883f5c651798e8.